### DISCUSSION TITLE
Home Assistant Support for Python 3.11.2 is being removed

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#2416] [Announcements]
![Screenshot 2024-02-01 10 20 54 PM](https://github.com/tteck/Proxmox/assets/21342670/89338500-d686-4766-b91c-918cab3ad8b2)

## The Home Assistant Core script has been updated to utilize Python 3.12.1

**Comment by dmshimself** [comment]
I think this might need some more dependencies installed.  I got errors in the HA log about not being able to install ha-av which is needed for the stream component.  After I manually installed the following, it all came good.  I had this issue ages ago with a non-tteck install and so what I did manually then might not be the minimum needed now, but it has given me a workaround

apt install libavdevice-dev libavformat-dev libavcodec-dev libavutil-dev libavfilter-dev

**Reply by tteck** [reply]
I added the dependencies, thanks.

**Reply by Yar1klamer** [reply]
Привет! Я новичок в home assistant, помоги пожалуйста, как добавить зависимости

**Comment by dmshimself** [comment]
:-)  Many thanks

**Comment by alexludwigklein** [comment]
Hi all, I followed above suggestion and installed the additional packages in the LXC. However, with python still being at version 3.11.2, the warning message still appears.
Best,
Alex

**Reply by tteck** [reply]
You need to create a new Home Assistant Core LXC and copy your data and MAC address over to the newly created LXC.

https://github.com/tteck/Proxmox/discussions/2341#discussioncomment-8125444

**Reply by alexludwigklein** [reply]
Thanks for the hint, I was not aware!
I guess, that means copying the content of ~/.homeassistant to the new LXC.
I see that while I'm typing this message, you are already linking the corresponding comment with instructions :)

**Reply by alexludwigklein** [reply]
Thanks again, it worked:
- installed new LXC
- copied data via script
- copied firewall and CT settings for USB devices
- copied MAC
- delete old LXC
- backup and restore new container under original CTID 
Cheers,
Alexander
